src/renderers/shaders/ShaderLib/meshphysical_vert.glsl

```diff
@@ -0,0 +1,53 @@
+#define STANDARD
```

Comment:
Wonder if this code could be shared with meshstandard_vert with a few exceptions if PHYSICAL is defined?